code/modules/mob/mob_defines.dm

```diff
@@ -54,6 +54,8 @@
 	var/eye_damage = 0//Living, potentially Carbon
 	var/lastpuke = 0
 
+	var/circulatory = 0 // Some simple mobs will receive damage from disabler bolts
```

Comment:
I think the issue is the var is rather snowflake and only serves to add yet another var to mobs?